CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update `canonical` to v0.6.0 [#44](https://github.com/dusk-network/dusk-pki/issues/44)
```

Comment:
Always mention from which version we're updating, both here and in the commit messages.